testsuite/forge-cli/src/main.rs

```diff
@@ -425,7 +425,8 @@ fn single_test_suite(test_name: &str) -> Result<ForgeConfig<'static>> {
                 helm_values["chain"]["epoch_duration_secs"] = 60.into();
             })),
         "state_sync" => config
-            .with_initial_fullnode_count(1)
+            .with_initial_validator_count(NonZeroUsize::new(4).unwrap())
```

Comment:
FYI we were defaulting to 30 validators previously https://github.com/aptos-labs/aptos-core/pull/3644/files#diff-72030b71c7c579ba040371a1574c390f94b62f40581abb29a3cd26cb4a5018dbR418. 4 validators and 4 VFNs sgtm tho

Reply:
Thanks @rustielin. Yeah, let's start small and scale if we need to